test/ifconfig.interfaces.test.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 var fixtures = require('./fixtures');
 var t = require('chai').assert;
+var path = require('path');
+var INTERFACE_FILE = path.resolve(__dirname, './tmp/interfaces');
```

Comment:
- this file does not seem to exist in the PR
- I don't see a test validating the parsing, am I right?

Reply:
Adds a Test validating the parsing from the interfaces file